packages/suite/src/hooks/wallet/trading/form/common/useTradingFiatValues.ts

```diff
@@ -71,53 +83,51 @@ export const useTradingFiatValues = ({
             currentTokenAddress?: TokenAddress,
         ): Promise<FiatRatesResult | null> => {
             if (!value) return null;
+            const tokenAddress = currentTokenAddress ?? contractAddress;
 
             const updateFiatRatesResult = await dispatch(
                 updateFiatRatesThunk({
                     tickers: [
                         {
                             symbol,
-                            tokenAddress: currentTokenAddress ?? tokenAddressTyped,
+                            tokenAddress: isNativeToken ? undefined : tokenAddress,
                         },
                     ],
                     baseCurrencyCode: value,
                     rateType: 'current',
                     fetchAttemptTimestamp: Date.now() as Timestamp,
+                    skipCache: true,
```

Comment:
Do you think that it makes sense to fetch this only if experiment is valid?

Probably not right? It is not going to be that many requests so probably good to leave it like this so that it behaves same all the time